ci/default.yml

```diff
@@ -45,6 +45,14 @@ test_tools_datatests_aarch64:
     - if: $BACKEND == 'embedded'
       variables:
         SLURM_TIMELIMIT: '00:15:00'
+    - if: $COMPONENT == 'driver' && $BACKEND == 'gtfn_gpu' && $LEVEL == 'integration'
+      variables:
+        # With more than 2 there are CUDA_ERROR_ILLEGAL_ADDRESS errors
+        NUM_PROCESSES: 2
+    - if: $COMPONENT == 'dycore' && $BACKEND == 'gtfn_gpu' && $LEVEL == 'integration'
+      variables:
+        # With more than 4 there are CUDA_ERROR_ILLEGAL_ADDRESS errors
+        NUM_PROCESSES: 4
```

Comment:
```suggestion
```
before merge. I can't see how this could fix `CUDA_ERROR_ILLEGAL_ADDRESS` (unless it's actually an oom issue)

Reply:
Running multiple tests in parallel one a single GPU has a memory high watermark of ~70GBs as far as I've seen. This together with the `pytest` stuff running multiple tests on a single GPU with MPS might generate the issue. The `CUDA_ERROR_ILLEGAL_ADDRESS` comes from a `cudaFree` somewhere at the end of the `gtfn` execution as far as I remember. I will investigate more once I have some time